cylc/uiserver/data_store_mgr.py

```diff
@@ -72,9 +72,9 @@ def __init__(self, workflows_mgr):
         self.delta_queues = {}
 
     def update_contact(self, w_id, contact_data=None):
-        delta = DELTAS_MAP[WORKFLOW]()
-        delta.time = time.time()
-        flow = delta.updated
+        delta = DELTAS_MAP[ALL_DELTAS]()
+        delta.workflow.time = time.time()
+        flow = delta.workflow.updated
```

Comment:
Trying to create a delta for a workflow but we can't use `DELTAS_MAP[WORKFLOW]`?

Reply:
`DELTAS_MAP[WORKFLOW]` is the individual workflow delta, but we create the `delta_store` out of the `AllDeltas` message using .. It doesn't really matter, because it will only contain the workflow data we set anyway..

https://github.com/cylc/cylc-flow/blob/eab1b1c8fea70b982b19bde6cce5570165bd74dc/cylc/flow/data_store_mgr.py#L199-L242